src/Microsoft.Language.Xml.Tests/TestModification.cs

```diff
@@ -13,5 +13,14 @@ public void TestEmptyElementWithContent ()
             root = root.WithContent (SyntaxFactory.SingletonList (content.AsNode));
             Assert.Equal("<a><b /></a>", root.ToFullString());
         }
+

```

Comment:
I added this test in order to demonstrate what is currently happening when we add content to an element with an attribute.   I included it in a separate commit in the case that you don't want to include it.